README.md

```diff
@@ -63,6 +64,11 @@ A design of the main view of a mobile application was made, which will be respon
 To get a local copy up and running, follow these steps.
 just visit my repo folder (https://github.com/rodrigopenayo98/my-rod-porf/tree/main-branch)
 
+
+## Live Demo
+
+To see the display of my portfolio use this link: (https://rodrigopenayo98.github.io/)

```

Comment:
- `[OPTIONAL]` Well done, made the changes asked by your previous code reviewer. Kindly add the section before the `Getting Started` because in your `README.md` table your *Live Demo* link is given before `getting started` but actually the `section` is after the `Getting Started` section. Actually, it makes your `README` a little bit unprofessional Please correct this mistake, rest everything is fine, you can take a look at the below image to understand things better visually.

### Live demo section in your `README`

![Screenshot from 2023-05-08 15-53-26](https://user-images.githubusercontent.com/66476329/236800873-91b9679c-e981-4a65-ab31-d56d850c0976.png)
 

### Actual live demo section looks like

![Screenshot from 2023-05-08 15-53-37](https://user-images.githubusercontent.com/66476329/236801027-fc4d8869-47f5-4038-abec-e6fce35172af.png)